src/components/JobDetail/JobDetail.tsx

```diff
@@ -1,5 +1,8 @@
+import { useJob } from '@/hooks/useJobs';
+import { useMatch } from 'react-location';
 function JobDetail() {
+  const jobId: string = useMatch().params.jobId;
+  const { data = null } = useJob(jobId);
```

Comment:
This seems like you are not doing anything with the result of useJob here. Maybe remove these changes and make this a different PR. We can discuss what this page should look like.

Reply:
Since you already worked on this, maybe just the data from JobDetails.